.github/workflows/test.yml

```diff
@@ -49,6 +53,8 @@ jobs:
 
       - name: Run tests
         env:
+          FLOW_WALLET_DATABASE_TYPE: "psql"
+          FLOW_WALLET_DATABASE_DSN: "postgresql://wallet:wallet@localhost:5432/wallet"
```

Comment:
Changed test workflow to use postgres since sqlite does not support `SELECT FOR UPDATE` feature.